package.json

```diff
@@ -1,35 +1,28 @@
 {
-  "name": "excel-custom-functions",
-  "version": "1.0.0",
-  "description": "Create Excel functions using JavaScript.",
-  "main": "customfunctions.js",
+  "name": "excel-functions",
+  "version": "0.0.2",
+  "main": "./src/customfunctions.ts",
+  "typings": "lib/index.d.ts",
   "scripts": {
-    "start": "webpack-dev-server --mode development --https --key ./certs/server.key --cert ./certs/server.crt --cacert ./certs/ca.crt --port 3000 --hot --inline",
-    "sideload": "office-toolbox sideload -m manifest.xml -a Excel",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack -p --mode production",
+    "sideload": "office-toolbox sideload -m manifest.xml -a excel",
+    "start": "start npm run webpack && start npm run dev-server && npm run sideload",
+    "dev-server": "webpack-dev-server --mode development --https --key ./certs/server.key --cert ./certs/server.crt --cacert ./certs/ca.crt --port 8081 --hotOnly",
+    "webpack": "webpack -p --mode development --watch"
```

Comment:
I recommend changing "webpack" to "watch", so that "npm run watch" runs the file watcher.